backends/platform/sdl/opendingux/miyoo.mk

```diff
@@ -0,0 +1,50 @@
+OD_EXE_STRIPPED := scummvm_stripped$(EXEEXT)
+bundle = sd-root
+
+all: $(OD_EXE_STRIPPED)
+
+$(OD_EXE_STRIPPED): $(EXECUTABLE)
+	$(STRIP) $< -o $@
+
+$(bundle): all
```

Comment:
I would try to refactor opendingux.mk to make it fit this folder hierarchy.
It seems that essentially, it's about adding some folder in the middle: this could be done through some variable (blank in generic case and defined to games/scummvm in miyoo case).

Reply:
It's not just this. Miyoo doesn't use .opk and instead it's a zip to unpack to the root. Big part of opendingux.mk is handling opk that we don't have here